### DISCUSSION TITLE
How to remove incorrectly assigned chiral centers?

### DISCUSSION THREAD

**Discussion by chengthefang** [rdkit/rdkit#5076] [Q&A] [answered]
Hi all,

I am currently dealing with a batch of molecules and find some of them have incorrectly assigned chiral centers. For example, it assigned the SP2 or SP3 Carbon a chiral center. I wonder if there is an automatic way to clean up the structure.

Here is an example. The molecule below should only have one chiral center, but the input file mistakenly assigned other bridged carbons as chiral center. How could I get rid of them? I tried "Chem.AssignStereochemistry" function and "Standardizer" function in molvs package but without any luck.  Any thoughts?

![image](https://user-images.githubusercontent.com/7895104/157573617-66d11cbf-6a9d-4327-841c-01ab8b1d4abe.png)

Thanks,
Cheng

**Comment by greglandrum** [comment, marked as answer]
@chf42 Those molecules don't have stereo assigned to sp2 carbons - the wedged bonds only affect that atom where they start, not the atom where they end.

In the case of the bridgehead atoms here: this is a tricky one where sometimes those can actually have stereochemistry associated with them and sometimes not. The RDKit has special rules for dealing with systems like this and it doesn't always get things right. This is an example where it's not getting it right.

You can remove the chiral markings from an atom manually by calling `SetChiralTag()` on the atom. Here's an example of doing that for your molecule:
```
In [38]: mol = Chem.MolFromSmiles("CC[C@H](C)NC(=O)[C@]12C[C@](C1)(C2)c1ncccn1")

In [39]: Chem.MolToSmiles(mol)
Out[39]: 'CC[C@H](C)NC(=O)[C@]12C[C@](c3ncccn3)(C1)C2'

In [40]: mol.GetAtomWithIdx(7).SetChiralTag(Chem.ChiralType.CHI_UNSPECIFIED)

In [41]: mol.GetAtomWithIdx(9).SetChiralTag(Chem.ChiralType.CHI_UNSPECIFIED)

In [42]: Chem.MolToSmiles(mol)
Out[42]: 'CC[C@H](C)NC(=O)C12CC(c3ncccn3)(C1)C2'
```

**Reply by chengthefang** [reply]
> @chf42 Those molecules don't have stereo assigned to sp2 carbons - the wedged bonds only affect that atom where they start, not the atom where they end.
> 
> In the case of the bridgehead atoms here: this is a tricky one where sometimes those can actually have stereochemistry associated with them and sometimes not. The RDKit has special rules for dealing with systems like this and it doesn't always get things right. This is an example where it's not getting it right.
> 
> You can remove the chiral markings from an atom manually by calling `SetChiralTag()` on the atom. Here's an example of doing that for your molecule:
> 
> ```
> In [38]: mol = Chem.MolFromSmiles("CC[C@H](C)NC(=O)[C@]12C[C@](C1)(C2)c1ncccn1")
> 
> In [39]: Chem.MolToSmiles(mol)
> Out[39]: 'CC[C@H](C)NC(=O)[C@]12C[C@](c3ncccn3)(C1)C2'
> 
> In [40]: mol.GetAtomWithIdx(7).SetChiralTag(Chem.ChiralType.CHI_UNSPECIFIED)
> 
> In [41]: mol.GetAtomWithIdx(9).SetChiralTag(Chem.ChiralType.CHI_UNSPECIFIED)
> 
> In [42]: Chem.MolToSmiles(mol)
> Out[42]: 'CC[C@H](C)NC(=O)C12CC(c3ncccn3)(C1)C2'
> ```

Thanks Greg! It makes perfect sense now. I think it might become tricky for some symmetric systems.

**Comment by jepdavidson** [comment]
@chf42 @greglandrum 

Hi Cheng, and Greg,

I just wanted to add a slightly different opinion into the mix - I don't think the RDKit is necessarily getting this "wrong".  But I think representing the bicyclopentane (BCP) bridgehead atoms as stereogenic is perhaps not what most people would expect (or, perhaps, want).

An analogy might be to think about cyclohexene:
![image](https://user-images.githubusercontent.com/7816101/157820910-71ff676e-4b3b-4391-a5ce-e159812f7b10.png)

It is not 'wrong' to explicitly represent it as (Z)-cyclohexene; but most people would take the (E) form to be unfeasibly high in energy - so most toolkits don't explicitly define the stereochemistry.

This changes once there is a feasible alternative - take, for example, cyclooctene:
![image](https://user-images.githubusercontent.com/7816101/157821855-2069a08b-f062-49b2-b0e5-0c2f4d2739a5.png)

I think the same thing is happening with bridged bicyclic ring systems.  In the case of BCP, the 'bridges' are single methylene groups - so there is only one feasible geometry.  But what if the bridges were made of 6 methylenes instead?

```
CC[C@H](C)NC(=O)[C@]12CCCCCC[C@](CCCCCC1)(CCCCCC2)c1ncccn1
```
![image](https://user-images.githubusercontent.com/7816101/157822675-b2720d25-c476-435e-8b9f-f3795704b33e.png)
3D view:
![image](https://user-images.githubusercontent.com/7816101/157824116-55aed402-78c6-4a1e-8e98-54a8f680f529.png)

```
CC[C@H](C)NC(=O)[C@]12CCCCCC[C@@](CCCCCC1)(CCCCCC2)c1ncccn1
```
![image](https://user-images.githubusercontent.com/7816101/157822788-027b2bab-48d0-42bc-85e5-4bec48082dff.png)
3D view:
![image](https://user-images.githubusercontent.com/7816101/157824225-ed901b7d-eb58-4595-98aa-9012e3bcbf17.png)

In the above case, it becomes clear that there are two valid geometries to distinguish between (they can be termed *cis* or *trans* across the 14-membered ring), and that is what the wedge/hash bonds are doing.

Apologies for the long reply(!) - to summarise, I think defaulting to explicit, and then applying a business rule to flatten the stereo representation (if that's what you want) is the most appropriate approach.

Kind regards

James
